src/test/java/emissary/test/core/junit5/LogbackTester.java

```diff
@@ -30,6 +36,30 @@ public LogbackTester(final String name) {
         logger.setAdditive(false);
     }
 
+    public void checkLogList(List<SimplifiedLogEvent> events) {
+        Validate.notNull(events, "Required: events != null");
+
+        assertEquals(events.size(), appender.list.size(), "Expected lengths do not match number of log messages");
+
+        for (int i = 0; i < appender.list.size(); i++) {
+            final ILoggingEvent item = appender.list.get(i);
+            final SimplifiedLogEvent event = events.get(i);
+            assertEquals(event.level, item.getLevel(), "Levels not equal for element " + i);
+            assertEquals(event.message, item.getFormattedMessage(), "Messages not equal for element " + i);
+            if (event.throwable.isEmpty()) {
+                assertNull(item.getThrowableProxy(), "Expected no exception for element" + i);
+            } else {
+                assertNotNull(item.getThrowableProxy(), "Expected an exception for element" + i);
+                Throwable expected = event.throwable.get();
+                IThrowableProxy proxy = item.getThrowableProxy();
+                assertEquals(expected.getClass().getName(), proxy.getClassName(), "Exception class name not equal for element " + i);
+                assertEquals(expected.getLocalizedMessage(), proxy.getMessage(), "Exception message not equal for element " + i);
+            }
+
+        }
+    }
+
+
     public void checkLogList(final Level[] levels, final String[] messages, final boolean[] throwables) {
```

Comment:
We _might_ want to decorate this legacy message with the `@Deprecated` annotation and direct developers to use the newer version. 
```suggestion
    /**
     * @deprecated Consider using the {@link #checkLogList(List)} overload instead of this version
     */
    @Deprecated    
    public void checkLogList(final Level[] levels, final String[] messages, final boolean[] throwables) {
```